templates/level-page.html

```diff
@@ -75,12 +75,10 @@ <h2 class="px-4 pt-2">{{_('commands')}}</h2>
                   <label for="keyword_toggle" class="flex flex-row items-center justify-center cursor-pointer"
                     data-cy="kwlang-switch-toggle">
                     <div class="relative flex w-full">
-                      <input id="keyword_toggle" type="checkbox" {% if ((g.keyword_lang != "en" and not
-                        request.args.get('keyword_language')) or (request.args.get('keyword_language') and
-                        request.args.get('keyword_language') != "en" )) %}checked{% endif %}
-                        onclick="hedyApp.toggle_keyword_language(`{% if ((g.keyword_lang != "en" and not
-                        request.args.get('keyword_language')) or (request.args.get('keyword_language') and
-                        request.args.get('keyword_language') != "en" )) %}en{% else %}{{g.lang}}{% endif %}`)"
+                        <input id="keyword_toggle" type="checkbox" {% if g.keyword_lang != "en" or
+                        request.args.get('keyword_language') != "en" %}checked{% endif %}
+                        _="on click hedyApp.toggle_keyword_language({% if g.keyword_lang != "en" or request.args.get('keyword_language') != "en" %}'en'
```

Comment:
`_="on click ..."`

hyperscript I guess (I havent used it yet), but it is already in use in other templates so it is nice.

I need to check the change in the conditions and test it again.

Reply:
If you are happy with the PR, can you click approve? (I am not sure you have the right rights?)

Do you see this option under Files Changed:

![Uploading image.png…]()

Reply:
> `_="on click ..."`
> 
> hyperscript I guess (I havent used it yet), but it is already in use in other templates so it is nice.
> 
> I need to check the change in the conditions and test it again.

Correct! We'll be using it in combination with htmx more often!

Reply:
@Felienne the first change was about spaces and quotes, now we have a change on the condition itself. I will test how it works but another eye on it would make it faster than me. (and yes, it is a bit horrifying, but I have the right to approve)

Reply:
nope, "removed unneded if statements" was a bad idea as it causes a new bug: upon changing levels, switcher always comes in checked as other language despite what is really set in page.

@hasan-sh it seems those extra ifs were checking this situation. Can you please put them back?